dev/breeze/src/airflow_breeze/commands/developer_commands.py

```diff
@@ -213,6 +213,18 @@ def shell(
 @option_mount_sources
 @option_integration
 @option_image_tag_for_running
+@click.option(
+    '--skip-asset-compilation',
+    help="Skips compilation of assets when starting airflow even if the content of www changed "
+    "(mutually exclusive with --dev-mode).",
+    is_flag=True,
+)
+@click.option(
+    '--dev-mode',
+    help="Starts webserver in dev mode (assets are always recompiled in this case when starting "
```

Comment:
```suggestion
    help="Starts webserver in dev mode (assets are always recompiled in this case when starting) "
```

Oh no, you dropped one!